wallet/local/accounts.go

```diff
@@ -0,0 +1,219 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package local
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/wallet/subnet/primary"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/account"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/network"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/utils"
+)
+
+// generateAccountName generates an automatic account name based on the current count
+func (w *LocalWallet) generateAccountName() string {
+	return fmt.Sprintf("account-%d", len(w.accounts)+1)
+}
+
+// setWalletAccount loads an account into the underlying primary wallet
+func (w *LocalWallet) setWalletAccount(ctx context.Context, acc account.Account, net network.Network) error {
+	wallet, err := getWalletFromAccount(ctx, acc, net, nil)
+	if err != nil {
+		return err
+	}
+	w.wallet = wallet
+	return nil
+}
+
+// Accounts returns all accounts managed by this wallet with their info
+func (w *LocalWallet) Accounts() map[string]account.AccountInfo {
```

Comment:
To be consistent, these getter functions should return a type account.Account. All functions that return something related to an Account should be returning account.Account not account.AccountInfo.